hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/writebehind/StoreWorker.java

```diff
@@ -71,6 +69,7 @@
      * @see #calculateHighestStoreTime
      */
     private long lastHighestStoreTime;
+    private boolean stopped;
```

Comment:
no need to make it `volatile` it is under `synchronized` blocks.
changed the name.